sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgInstructionFactory.cs

```diff
@@ -415,6 +401,26 @@ private IEnumerable<Instruction> ProcessAssignmentExpression(AssignmentExpressio
 
             var rightExpression = expressionService.GetExpression(assignmentExpression.Right);
 
+            // Handle the special add assignment (+=)
+            // In this block we will only handle the concatenation part, not the assignment (that will be handled right after)
+            if (assignmentExpression.IsKind(SyntaxKind.AddAssignmentExpression))
+            {
+                var addOperatorSymbol = GetSymbol<IMethodSymbol>(assignmentExpression);
+
+                // TODO: Handle implicit ToString
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this 'TODO' comment. ([csharpsquid:S1135](https://sonarcloud.io/organizations/sonarsource/rules#rule_key=csharpsquid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=sonaranalyzer-dotnet&pullRequest=1573&issues=AWSonkbrMDLKieESTzU4&open=AWSonkbrMDLKieESTzU4)

[ISSUE_KEY]: AWSonkbrMDLKieESTzU4